rotkehlchen/tests/unit/test_ledger_actions.py

```diff
@@ -26,8 +26,13 @@ def test_serialize_deserialize_for_db():
 
 def test_all_action_types_writtable_in_db(database, function_scope_messages_aggregator):
     db = DBLedgerActions(database, function_scope_messages_aggregator)
+
+    # Query to check that the changes have been committed to the db
```

Comment:
nitpick: This comment is not really needed. It's obvious. Remove it from here and rest of the tests